code/game/objects/items/rogueweapons/melee/swords.dm

```diff
@@ -82,6 +82,22 @@
 	swingdelay = 0
 	item_d_type = "stab"
 
+/obj/item/rogueweapon/sword/stone
+	force = 15 //Weaker than a short sword
+	force_wielded = 20 // Less than other swords
```

Comment:
this makes it stronger than the short sword for most characters, and stronger than most iron and steel knives and daggers. a force of 15 and no wieldability is fine.